recipes-graphics/westeros/westeros.bb

```diff
@@ -2,10 +2,11 @@ require westeros.inc
 
 SUMMARY = "This receipe compiles the westeros compositor component"
 
-SRC_URI += "file://0001-Use-intptr_t-to-avoid-precision-errors-on-aarch64.patch \
+SRC_URI += " \
```

Comment:
Adding file this way is convenient, but it causes warnings. Let's keep that in mind for later

Reply:
I am not seeing any warning due to this. will revert, if I get it